code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -6,7 +6,7 @@
 	item_state = "laser"
 	fire_sound = 'sound/weapons/Laser.ogg'
 	slot_flags = SLOT_BELT|SLOT_BACK
-	w_class = 3
+	w_class = 4

```

Comment:
w_class 4 can fit in bags, and has been able to since the inventory rework that occurred awhile ago.  w_class 5 can't fit in bags, which the LWAP/PTR-7 has.